scss/components/_navbar.scss

```diff
@@ -396,7 +396,7 @@
     line-height: $navbar-brand-height;
     margin: 0;
     padding: 0;
-    padding-left: px(12.5);
+    padding-left: 12.5px;
```

Comment:
Using floating point numbers in CSS can have unpredictable results since browsers have different ways of rounding up things.